src/app/parcels/form/ParcelForm.tsx

```diff
@@ -56,7 +56,7 @@ import ListTypeCard from "./formSections/ListTypeCard";
 
 export interface ParcelFields extends Fields {
     clientId: string | null;
-    listType: ListType | null;
+    listType: ListType | undefined;
```

Comment:
Purely for style convention, we tend to write optional properties with `?`, which means "or undefined":
`listType?: ListType;`
Can we change that here and in the other place?